lib/rules/resolve-async.js

```diff
@@ -119,3 +119,9 @@ module.exports = function (context) {
         }
     };
 };
+
+module.exports.schema = [
+    {
+        "enum": ["except-parens", "always"]
+    }
+];
```

Comment:
Uhhh... I guess you can see that I put the `schema` in the wrong place... any good way to test schemas? I couldn't find examples in the official ESLint rules and it's probably not a deal breaker.